.github/PULL_REQUEST_TEMPLATE/simple_pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- This template should only be used in rare situations. If it is possible that the details will at all effect the ability of other contributors, the detailed pull request template should be used. If your reviewer believes more info is necessary, they may have you complete the detialed_pull_request_template. -->
+
+## Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests.
```

Comment:
```suggestion
- [ ] I have added thorough tests. <-- Skip if a documentation update -->
```

Reply:
Tests of code in general should be required.

Reply:
got it!